index.html

```diff
@@ -119,124 +120,124 @@ <h2 class="area-of-control__heading">I'm an interesting person!</h2>
-              <figure class="catchphrase">
-                <div class=catchphrase__blackjack">
-                  <p class="catchphrase__xxx">
+            <div class="media__right grid__item--quarter">
+              <figure class="quote">
+                <div class="quote__card">
+                  <p class="quote__text">
                     I bought a car. Turned out to be an alien robot.
                   </p>
                 </div>
                 <figcaption class="catchphrase__zzz">
-                  <span><img class="catchphrase__yyy" src="https://i.kinja-img.com/gawker-media/image/upload/s--hw7ssOEj--/i6yafkokqsxxazd95fis.png" alt=""></span>
+                  <span><img class="quote__pic" src="https://i.kinja-img.com/gawker-media/image/upload/s--hw7ssOEj--/i6yafkokqsxxazd95fis.png" alt=""></span>
 
-                  <div class="side-by-side">
-                    <h4 class="catchphrase__character">Shia LaBeouf</h4>
-                    <p class="catchphrase__show"Transformers</p>
+                  <div class="text-stack">
+                    <h4 class="quote__attribution">Shia LaBeouf</h4>
+                    <p class="quote__title">Transformers</p>
                   </div>
                 </figcaption>
               </figure>
             </div>
 
-            <div class="leela__thing leela__thing--forth">
-              <figure class="catchphrase">
-                <div class="catchphrase__blackjack"
-                  <p class="catchphrase__xxx">
+            <div class="media__right grid__item--quarter">
+              <figure class="quote">
+                <div class="quote__card">
+                  <p class="quote__text">
                     I bought a car. Turned out to be an alien robot.
                   </p>
-
+                </div>
                 <figcaption class="catchphrase__zzz">
-                  <span><img class="catchphrase__yyy" src="https://i.kinja-img.com/gawker-media/image/upload/s--hw7ssOEj--/i6yafkokqsxxazd95fis.png" alt=""></span>
+                  <span><img class="quote__pic" src="https://i.kinja-img.com/gawker-media/image/upload/s--hw7ssOEj--/i6yafkokqsxxazd95fis.png" alt=""></span>
 
-                  <div class="side-by-side">
-                    <h4 class="catchphrase__character">Shia LaBeouf</h4>
-                    <p class="catchphrase__show">Transformers</p>
+                  <div class="text">
+                    <h4 class="quote__attribution">Shia LaBeouf</h4>
+                    <p class="quote__title">Transformers</p>
                   </div>
                 </figcaption>
               </figure>
             </div>
           </div>
         </div>
       </div>
-
-      <div class="area-of-control area-of-control__photos" id="photos">
-        <div class="box">
-          <div class="leela">
-            <div class="leela__thing">
-              <div class="blackjack">
-                <div class="blackjack__photo">
+    <div class="section">
```

Comment:
Check your classes on line 204. It looks like you may have added an extra opening div?